### PR TITLE
Duplicated SOAP dependency in the jdk 11 commit

### PR DESCRIPTION
**Pull request by desmax74**
@mariofusco 
Removed duplicated soap dependency added in this commit: https://github.com/kiegroup/droolsjbpm-build-bootstrap/commit/7033cb909f337cb566cbec4c70f8f65058a96d6c

### CODE REVIEW

**Comment by mariofusco** [comment]
jenkins execute full downstream build

File: pom.xml [hunk 1]
```diff
@@ -2700,37 +2706,32 @@
...
         <groupId>javax.xml.bind</groupId>
         <artifactId>jaxb-api</artifactId>
-        <version>2.3.1</version>
+        <version>${version.javax.xml.bind}</version>
       </dependency>
       <dependency>
         <groupId>javax.jws</groupId>
         <artifactId>javax.jws-api</artifactId>
-        <version>1.1</version>
+        <version>${javax.jws-api}</version>
```
Comment (mbiarnes) [inline pom.xml]: @desmax74 here is missing ${version.javax.jws-api}

Comment (desmax74) [reply pom.xml]: Fixed

**Comment by desmax74** [comment]
jenkins execute full downstream build

**Comment by desmax74** [comment]
jenkins execute full downstream build

File: pom.xml [hunk 2]
```diff
@@ -248,7 +248,7 @@
 
     <version.org.jboss.spec.javax.websocket>1.1.1.Final</version.org.jboss.spec.javax.websocket>
 
-    <version.javax.xml.soap>1.3.5</version.javax.xml.soap>
```
Comment (mbiarnes) [inline pom.xml]: this upgrade to 1.4.0 can be removed since it is already in kie-parent

File: pom.xml [hunk 3]
```diff
@@ -264,6 +264,12 @@
     <version.org.mvel>2.4.3.Final</version.org.mvel>
```
Comment (mbiarnes) [inline pom.xml]: org.mvel can be removed since it is already in jboss-ip-bom

File: pom.xml [hunk 4]
```diff
@@ -264,6 +264,12 @@
...
     
     <version.ch.obermuhlner>2.0.1</version.ch.obermuhlner>
+    <version.javax.xml.bind>2.3.1</version.javax.xml.bind>
```
Comment (mbiarnes) [inline pom.xml]: javax.xml.bind can be removed as it was already removed - and it is not anymore in kie-parent

File: pom.xml [hunk 5]
```diff
@@ -264,6 +264,12 @@
...
+    <version.javax.jws-api>1.1</version.javax.jws-api>
+    <version.javax.xml.ws>2.3.1</version.javax.xml.ws>
+    <version.com.sun.xml.ws>2.3.1</version.com.sun.xml.ws>
```
Comment (mbiarnes) [inline pom.xml]: version.javax.xml.ws can be removed as it is already removed in kie-parent

**Comment by mbiarnes** [comment]
@desmax74 @mareknovotny 
once this PR is green it can be merged and the PR https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/902 should be closed!

**Comment by yurloc** [comment]
Just for the record: this PR needed a full downstream build. The downstream build would have failed in `jbpm` repository and would have prevented breaking kiegroup build **again**.

Not blaming anyone. It's not obvious there still might be problem with a PR when its PR build is green. Maybe the PR age should have raised a red flag asking for more attention. Anyway, lesson to learn: always execute full downstream build when *removing* dependencies from `kie-parent`.

**Pull request merged**